tic-tac-toe.js

```diff
@@ -1,7 +1,121 @@
+        if (game.checkIfWon(board,"player_2")) {
+          alert(game._current_player + " won!")
+          game.reset(board,game)
+          updateMessage("Your turn, " + game._current_player)
+        } else {
+          if (game.checkIfDraw(board)) {
+            alert("It's a draw!")
+            game.reset(board,game)
+            updateMessage("Your turn, " + game._current_player)
+          }
+        game.changePlayer()
+        updateMessage("Your turn, " + game._current_player)
+        }
+      } else {
+        space.addClass("player_1")
+        if (game.checkIfWon(board,"player_1")) {
+          alert(game._current_player + " won!")
+          game.reset(board,game)
+          updateMessage("Your turn, " + game._current_player)
+        } else {
+          if (game.checkIfDraw(board)) {
+            alert("It's a draw!")
+            game.reset(board,game)
+            updateMessage("Your turn, " + game._current_player)
+          }
+        game.changePlayer()
+        updateMessage("Your turn, " + game._current_player)
+        }
+      }
+    }
+  })
+})
+
+
 function TicTacToe() {
+  this._current_player = "Player 1"
+  this._message = $('#message').text("Your turn, " + this._current_player)
 
-}
+  this.changePlayer = function() {
+    if (this._current_player === "Player 1") {
+      this._current_player = "Player 2"
+    } else {
+      this._current_player = "Player 1"
+    }
+  }
+
+  this.isSpaceEmpty = function(td) {
+    if (td.hasClass('player_2') || td.hasClass('player_1')) {
+      return false
+    } else {
+      return true
+    }
+  }
+
+  this.checkIfWon = function(board, player) {
+    var won = false
+    if(board.find('#space-1').hasClass(player) && board.find('#space-2').hasClass(player) && board.find('#space-3').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-1').hasClass(player) && board.find('#space-4').hasClass(player) && board.find('#space-7').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-1').hasClass(player) && board.find('#space-5').hasClass(player) && board.find('#space-9').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-4').hasClass(player) && board.find('#space-5').hasClass(player) && board.find('#space-6').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-7').hasClass(player) && board.find('#space-8').hasClass(player) && board.find('#space-9').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-2').hasClass(player) && board.find('#space-5').hasClass(player) && board.find('#space-8').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-3').hasClass(player) && board.find('#space-6').hasClass(player) && board.find('#space-9').hasClass(player)) {
+      won = true
+    } else if (board.find('#space-3').hasClass(player) && board.find('#space-5').hasClass(player) && board.find('#space-7').hasClass(player)) {
+      won = true
+    }
```

Comment:
`checkIfWon` does what it needs to do! Is there a way to make this more concise to get rid of all of the `else if` statements? But it might not be worth it, because it's not like you're going to make this a larger game. It will only ever be these 8 statements. Also, since only one of the conditions will be true at a time, they could all just be `if` statements.